app/javascript/schools/courses/components/curriculum_editor/CurriculumEditor__ContentBlockCreator.re

```diff
@@ -144,6 +144,8 @@ let fileInputId = aboveContentBlock =>
   aboveContentBlock |> elementId("markdown-block-file-input-");
 let imageInputId = aboveContentBlock =>
   aboveContentBlock |> elementId("markdown-block-image-input-");
+let videoInputId = aboveContentBlock =>
+  aboveContentBlock |> elementId("markdown-block-video-input-");
```

Comment:
I think we should use the word `vimeo` instead of `video` in this file, since the implementation is very much tied with a specific provider. This goes for the icon as well.